### ISSUE TITLE
Aux window workspace file explorer selects wrong file

### ISSUE THREAD

**Issue by TomSwiedler** [microsoft/vscode#200452]
Type: <b>Bug</b>

1. Open workspacewith multiple folders
2. Open a file from the primary folder on the primary window
3. Open a file from a secondary folder on a secondary window
4. Focus on the secondary window file
5. Click on another file in the secondary folder tree
6. The folder list jumps to the primary folder and selects a file from that folder

VS Code version: Code 1.85.0 (af28b32d7e553898b2a91af498b1fb666fdebe0c, 2023-12-06T20:48:09.019Z)
OS version: Windows_NT x64 10.0.19045
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-6200U CPU @ 2.30GHz (4 x 2400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|7.88GB (1.94GB free)|
|Process Argv|--disable-extensions --crash-reporter-id 4bc38c91-4e66-4bff-a24f-c024707098e1|
|Screen Reader|no|
|VM|0%|
</details>Extensions disabled<details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30410667
py29gd2263:30899288
vsclangdf:30486550
c4g48928:30535728
dsvsc012cf:30540253
azure-dev_surveyone:30548225
3biah626:30602489
f6dab269:30613381
vscrp:30673768
showlangstatbar:30737416
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
welcomedialogc:30910334
pythonnosmt12:30797651
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
dsvsc013:30795093
dsvsc014:30804076
dsvsc015:30845448
pythontestfixt:30902429
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
h48ei257:30885898
pythontbext0:30879054
accentitlementsc:30887149
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
aa_t_chat:30882232

```

</details>

**Comment by gjsjohnmurray** [comment]
> 5\. Click on another file in the secondary folder tree

Please clarify what you mean by 'secondary folder tree'. Maybe paste a screenshot.

**Comment by TomSwiedler** [comment]
Step 0: Create a workspace with two folders and a couple of files in each:
![Step0](https://github.com/microsoft/vscode/assets/38703985/a666814f-18e5-459e-967d-edde4fa1611c)

Step 1: Open a file from primary folder in the primary window. Open a file from secondary folder in a secondary window. Then arrange the file tree so that only part of the tree is visible and click in the primary window editor. This will scroll the file tree to show the filename in the tree.

![Step1](https://github.com/microsoft/vscode/assets/38703985/ccd9bff3-5ad5-46dc-b49f-841d45fc4e7e)

Step 2: Click in the editor for the secondary (AUX) window. This will scroll the file tree to show the file in the secondary folder:


![Step2](https://github.com/microsoft/vscode/assets/38703985/110424a4-6b2d-47f2-ae29-873367474cf0)

Step 3: Click on the second file in the secondary folder. This will cause the file tree to scroll, and the click will be applied to the folder (or some other file) instead of the selected file.

![Step3](https://github.com/microsoft/vscode/assets/38703985/c8c5062f-5b2c-4875-b0e2-d4e28bb2deb6)

You can repeat the sequence of clicking in the primary editor and clicking in the Aux window editor and then clicking on the second file in the secondary folder and the issue will repeat.

**Comment by bpasero** [comment]
I think this will be hard to prevent: when you click into the first window, the editor of that window becomes active to the tree scrolls to reveal it. Your click is then no longer at the target you intended to.

If you disable `explorer.autoReveal` in settings, its probably fine.

**Comment by lramos15** [comment]
Yeah I don't know if we will be able to fix this or we call it as designed with the auto reveal setting
